spk/sickrage/src/wizard/install_uifile_nld

```diff
@@ -0,0 +1,22 @@
+[{
+    "step_title": "Installatie SickRage",
+    "items": [{
+        "type": "textfield",
+        "desc": "Klik op volgende om SickRage te installeren. (Of vul een alternative URL in.)",
```

Comment:
`"desc": "Klik op volgende om SickRage te installeren. (Of vul een alternatieve URL in.)",`